tests/suites/model/destroy.sh

```diff
@@ -0,0 +1,46 @@
+# Tests if Juju tracks the model properly through deletion.
+#
+# In normal behavior Juju should drop the current model selection if that
+# model is destroyed. This will fail if Juju does not drop it's current
+# selection.
+run_model_destroy() {
+	# Echo out to ensure nice output to the test suite.
+	echo
+
+	# The following ensures that a bootstrap juju exists.
+	file="${TEST_DIR}/test-model-destroy.log"
+	ensure "model-destroy" "${file}"
+
+	echo "Ensure current model is 'model-destroy'"
+	juju models --format json | jq -r '."current-model"' | check model-destroy
+
+	echo "Add new model 'model-new'"
+	juju add-model model-new
+
+	echo "Ensure current model is 'model-new'"
+	juju models --format json | jq -r '."current-model"' | check model-new
+
+	echo "Destroy model 'model-new'"
+	destroy_model "model-new"
+	juju switch :model-destroy
```

Comment:
We technically don't need the colon here

And also, this is incorrect I believe. The original test asserts that if the active model is destroyed, you are automatically switched to another

Reply:
I agreed about the colon. But I've concerns about automatic switching: 
https://pastebin.canonical.com/p/99Y9R7jQ7b/

Here is the origin (python-based) test: https://github.com/juju/juju/blob/137a772ed339b73b856e9adc0a5624976c2890b2/acceptancetests/assess_destroy_model.py#L47-L53